lib/utils/check-root-user.js

```diff
@@ -2,20 +2,26 @@
 
 const os = require('os');
 const chalk = require('chalk');
+const fs = require('fs');
 
 function checkRootUser() {
     // Skip if we're on windows
-    if (os.platform() !== 'linux') {
+    if (os.platform() !== 'linux' || process.getuid() !== 0) {
         return;
     }
 
-    if (process.getuid() === 0) {
+    if (fs.existsSync('/root/.digitalocean_password')) {
+        // We have a Digitalocean one click installation
+        console.error(`${chalk.yellow('We discovered that you are using the Digitalocean 1-click install.')}
+You need to create a user with regular account privileges and migrate your installation to use this user.
+Please follow the steps here ${chalk.underline.green('https://docs.ghost.org/docs/troubleshooting#section-fix-root-user')} to fix your setup.`);
```

Comment:
When i run `find . -group root -user root -exec chown <user>:<user> {} \;
` on my one-click DO, my `current folder` is still owned by root?


> lrwxrwxrwx 1 root  root    29 Jan 24 16:59 current -> /var/www/ghost/versions/1.6.2/

Reply:
Probably because it's a symlink.

Reply:
yeah - I think that's fine - although you might test running `ghost update` on a DO 1-click install to see if it's able to recreate the symlink after fixing the install.

Reply:
@AileenCGN can you test that pls? i destroyed my droplet already